paddle/fluid/pir/dialect/operator/ir/ops.yaml

```
@@ -1342,6 +1342,16 @@
     data_type : dtype
     backend : place
 
+- op : rank_attention
+  args : (Tensor x, Tensor rank_offset, Tensor rank_param, int max_rank = 3, int max_size = 0)
+  output : Tensor(input_help), Tensor(out), Tensor(ins_rank)
+  infer_meta :
+    func : RankAttentionInferMeta
+  kernel :
+    func : rank_attention
+    data_type : x
+  backward : rank_attention_grad
+
```

```suggestion
optional : ins_rank, input_help
```
旧IR下这两个output加了AsDispensable，这里在新IR下需要加入optional。